### ISSUE TITLE
Site editor: the 'more' Actions button in the navigation panel should be last

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#59280]
### Description

Follow-up to https://github.com/WordPress/gutenberg/issues/52590

In https://github.com/WordPress/gutenberg/issues/52590 an insonsistency with the 'Actions' and 'Edit' buttons palcement in the navigation panel was fixed. Reusing a screenshot from https://github.com/WordPress/gutenberg/issues/52590

<img width="750" alt="order" src="https://github.com/WordPress/gutenberg/assets/1682452/e6ae6d4c-41cc-46b5-8005-e7b80ab706a9">

Not the order of the buttons is consistent and it:
- First: the 'more' Actions button
- Last: the 'pencil' Edit button

However the logical concept of a 'More' button is that it is _preceded_ by other controls and then it provides 'more' stuff. As such, the logical expectation is to have the 'More' button _as the last thing_ in a group of controls.

All the other software I can think of htat use a 'More' button, they place is at the end of a controls group, as last thing.

Worth noting the actual label of this button is 'Actions' but it does use the ellipsis icon, which is pretty familiar convention for 'more'.

For better consistency with patterns users are familiar with and to match a logical expectation, a button that groups 'more' controls should always be last in the grop of controls it lives in.

I'd like also to note that the logical concept of 'more' implies that there are other controls before the more. In many places in the editor the ellipsis button is used 'in isolation' instead, with no other controls before it. Which Id argue it doesn;t make much sense, but that's another issue and should be addressed separately.

For now, the scope of this issue is to reverse the order of this button pair in the navigation panelG

### Step-by-step reproduction instructions

- Go to the Site editor and observe the order of the buttons in the following screens:
- Templates > A custom template.
- Templates > A default template that hae been customized.
- Pages > Any page
- Navigation > Any menu

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by afercia** [comment]
The navigation panel in the site editor has gone through several changes. I'm not sure there's a case where the ellipsis button is shown together with other buttons any longer. @WordPress/gutenberg-design can you confirm that? when you have a chance? If true, I think this issue can be closed.

**Comment by jameskoster** [comment]
I believe you are correct. I'm unable to locate any instances matching the screenshots in the OP. cc @getdave for good measure, iirc you worked closely on this panel?

I'll close, but please feel free to re-open if necessary.

**Comment by getdave** [comment]
I was also unable to replicate.
